draft-demarco-oauth-status-attestations.md

```diff
@@ -465,6 +465,35 @@ allowing Verifiers to examine and make educated conclusions based on a variety o
+
+## Privacy Consideration: Opacity of Status Attestation Content
+
+Another key privacy consideration is the design of the status attestation to ensure that it does not contain any information regarding the user or the holder of the digital credential. The status attestation is structured to provide only the necessary information to verify the non-revocation status of a digital credential without disclosing the identity of the holder or any personal information. This is achieved by focusing the attestation content on the credential's status and the verification process, rather than on the credential's holder. This design choice is instrumental in preserving the anonymity of the user, ensuring that the status attestation can be used in a wide range of verification scenarios without compromising the privacy of the individuals involved.
+
+## Unlinkability between Verifiers
+
+Status Attestations enhance privacy by ensuring that verifiers operate independently of each other without the need to interact with or disclose information to third-party entities or other verifiers. This design choice is crucial for maintaining unlinkability between verifiers, meaning that actions performed by one verifier cannot be correlated or linked to actions performed by another verifier. This independence is achieved by enabling verifiers to validate the status of a digital credential directly through the Status Attestation, without external communication. Such a mechanism is instrumental in preserving the privacy of the individuals whose credentials are being verified, as it minimizes the risk of tracking or profiling based on verification activities across different services.
+
+## Untrackability by Digital Credential Issuers
+
+Another cornerstone of the privacy-preserving features of Status Attestations is the prevention of tracking by digital credential issuers. Unlike models that require verifiers to query a central status list or the issuer directly, Status Attestations contain all necessary information for verification within the attestation itself. This approach ensures that the credential issuer cannot monitor when or where a digital credential is verified, thus significantly enhancing the privacy of the credential holder. By eliminating the need for real-time communication with the issuer for status checks, Status Attestations ensure that verification activities remain untrackable by the issuer, further solidifying the system's commitment to user privacy.
```

Comment:
This is typically called the "phone home" problem.

Reply:
Done